src/containers/Accounts/AccountTransactionTable/test/AccountTransactionTable.test.tsx

```diff
@@ -19,15 +20,13 @@ describe('AccountTransactionsTable container', () => {
   })
 
   const createWrapper = (
-    getAccountTransactionsImpl = () =>
-      new Promise(
-        () => {},
-        () => {},
-      ),
-    currencySelected = null,
+    getAccountTransactionsImpl = () => new Promise(() => {}),
+    currencySelected?: string,
     state = { hasToken: false },
   ) => {
-    getAccountTransactions.mockImplementation(getAccountTransactionsImpl)
+    ;(getAccountTransactions as Mock).mockImplementation(
```

Comment:
This leading `;` is confusing syntax to me

Reply:
That was done by the linter.